# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/cssom-view/scroll-behavior-smooth-positions.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/cssom-view/scroll-behavior-smooth-positions.html.ini
--- a/testing/web-platform/meta/css/cssom-view/scroll-behavior-smooth-positions.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/cssom-view/scroll-behavior-smooth-positions.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -5,9 +5,8 @@
 
   [Scroll positions when performing smooth scrolling from (0, 0) to (500, 250) using scroll() ]
     expected:
-      if not swgl and webrender and (os == "win") and debug: ["PASS", "FAIL"]
-      if not swgl and webrender and (os == "linux") and not fission: ["PASS", "FAIL"]
-      if swgl: ["PASS", "FAIL"]
+      if webrender and (os == "linux") and not fission: ["PASS", "FAIL"]
+      if webrender and (os == "win") and debug: ["PASS", "FAIL"]
 
   [Scroll positions when performing smooth scrolling from (1000, 500) to (500, 250) using scrollBy() ]
     expected:
@@ -15,6 +14,9 @@
 
   [Scroll positions when performing smooth scrolling from (0, 500) to (500, 250) using scrollIntoView() ]
     expected:
-      if not swgl and (os == "win") and debug and webrender: ["PASS", "FAIL"]
+      if (os == "win") and debug and webrender: ["PASS", "FAIL"]
+
+  [Scroll positions when performing smooth scrolling from 0 to 250 by setting scrollTop ]
+    expected:
       if swgl and (os == "win"): ["PASS", "FAIL"]
 